package.json

```diff
@@ -7,7 +7,7 @@
       "connected-react-router": "^6.4.0",
       "fetch-send-json": "0.0.2",
       "gh-pages": "^2.0.1",
-      "nearlib": "^0.6.2",
+      "nearlib": "^0.8",
```

Comment:
does this default to 0.8.0?

Reply:
according to this https://semver.npmjs.com it should match versions 0.8.0 - 0.8.2 (if you take current published)

I am still kind of confused whether this will update to latest minor version by itself, or only when any of other dependencies depends on it.

Reply:
Resolving, cause now it already says 0.8.2 anyway